frontend/public/locales/en/gcweb.json

```diff
@@ -87,6 +87,13 @@
     "page-message": "We're sorry you ended up here. Sometimes a page gets moved or deleted, but hopefully we can help you find what you're looking for.",
     "page-link": "Return to the <home>home page</home>"
   },
```

Comment:
Can we remove all the `gcweb:not-found.*` i18n keys since we have replaced them with `gcweb:public-not-found.*` and `gcweb:protected-not-found.*`?

Reply:
I tried that, but got an error.

On investigation, I found that it was still used in [$.tsx](https://github.com/DTS-STN/canadian-dental-care-plan/blob/main/frontend/app/routes/%24lang/%24.tsx#L21)

Reply:
The `i18nNamespaces`, `pageIdentifier` and the `<NotFoundError />` in [$.tsx](https://github.com/DTS-STN/canadian-dental-care-plan/blob/main/frontend/app/routes/%24lang/%24.tsx) are all referencing that of the public not found page. I wonder if it would make sense to reference the document title for the public not found page too.

At the very least we should be able to delete all the i18n keys that are not `document-title`.

Reply:
Put that way I think it makes sense for [$.tsx](https://github.com/DTS-STN/canadian-dental-care-plan/blob/main/frontend/app/routes/%24lang/%24.tsx) to use the public-not-found.

I have implemented that change in commit c8692c6e39486340fdb76cb769fee6041d3ee882.